src/manifold/src/impl.cpp

```diff
@@ -478,23 +480,17 @@ Manifold::Impl::Impl(const Mesh& mesh, const MeshRelationD& relation,
   Finish();
 }
 
-void Manifold::Impl::Hull(const std::vector<glm::vec3>& vertPos) {
+void Manifold::Impl::Hull(const std::vector<vec3>& vertPos) {
   size_t numVert = vertPos.size();
   if (numVert < 4) {
     status_ = Error::InvalidConstruction;
     return;
   }
 
-  Vec<glm::dvec3> pointCloudVec(numVert);
-  manifold::transform(vertPos.begin(), vertPos.end(), pointCloudVec.begin(),
-                      [](const glm::vec3& v) { return glm::dvec3(v); });
-  QuickHull qh(pointCloudVec);
-  ConvexHull hull = qh.getConvexHullAsMesh(pointCloudVec, false);
-  // TODO : Once double PR lands, replace this with move
-  vertPos_.resize(hull.vertices.size());
-  manifold::transform(hull.vertices.begin(), hull.vertices.end(),
-                      vertPos_.begin(),
-                      [](const glm::dvec3& v) { return glm::vec3(v); });
+  // Vec<vec3> pointCloudVec=vertPos;
+  QuickHull qh(vertPos);
+  ConvexHull hull = qh.getConvexHullAsMesh(vertPos, false);
+  vertPos_ = std::move(hull.vertices);
```

Comment:
FYI @Kushal-Shah-03 I did some cleanup on your Hull here as part of the move to doubles. Hopefully we can merge this soon.

Reply:
Thanks for the update, that's quite exciting, if you face issues with passing the hull tests ping me, I'll try to see if I can help.